RInAppMessaging/Classes/Router.swift

```diff
@@ -64,7 +70,7 @@ internal class Router: RouterType {
             }
 
             DispatchQueue.main.async {
-                guard let view = viewConstructor?() else {
+                guard let view = viewConstructor?(), confirmation() else {
```

Comment:
Would `confirmation() == true` be clearer?

Reply:
I wanted to put a name for this closure so it wouldn't be necessary but I couldn't find a better name. Probably `confirmation() == true` is clearer